hosts_example.yml

```diff
@@ -189,10 +189,25 @@ all:
     # rbac_component_additional_system_admins:
     #   - user1
     #   - user2
+    ##
+    ####  Configuring Role Based Access Control with a remote MDS ####
+    ## To have CP-Ansible configure Brokers and Components for RBAC with the MDS on a remote cluster, set these mandatory variables in addition to those listed above:
+    # rbac_enabled: true
+    # external_mds_enabled: true
+    # external_super_user: mds
```

Comment:
whats the difference between external_super_user and mds_super_user in our code?

Reply:
@domenicbove I actually don't think we need the external_super_user var, its from the one pager and while conducting the work I've realized its not required.  As long as you set the mds_super_user var in your inventory when setting up your second cluster, it all works as expected.  It could be the same as the first cluster or it could be different, it doesn't really matter from our perspective.

Reply:
if var is not used in code please remove from this file and elsewhere (it may have gotten into molecule.yml files)

Reply:
Sorry, maybe I wasn't clear in my last comment, I've already removed it due to it not being needed.